<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>276</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>618</width>
    <height>276</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>618</width>
    <height>276</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Config Parser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/CoPUI.ico</normaloff>../Icons/CoPUI.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Folder</string>
   </property>
   <widget class="QPushButton" name="openB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open folder</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="configL">
       <property name="text">
        <string>Config file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="configT">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit</string>
   </property>
   <widget class="QListWidget" name="keyE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>411</width>
      <height>181</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>411</width>
      <height>181</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="changeB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
